.github/workflows/clang-tidy-review.yml

```diff
@@ -0,0 +1,72 @@
+---
+name: clang-tidy-review
+
+on:
```

Comment:
Not sure if this is a YAML thing, but I'm having a hard time correctly identifying the identation because it uses two spaces. If possible, I'd prefer to have 4 spaces.

Reply:
I believe it is a GtiHub Yaml thingy. Not sure will check what happens when using four spaces